ci-operator/step-registry/ovn/conf/ovn-conf-commands.sh

```diff
@@ -3,13 +3,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+set +o errexit
+NETWORKING_BLOCK=$(cat ${SHARED_DIR}/install-config.yaml | grep "networking\:")
+set -o errexit
+if [ ! -z ${NETWORKING_BLOCK} ]; then 
+  cat "${SHARED_DIR}/install-config.yaml" | sed 's/networking\:/networking\:\n  networkType: OVNKubernetes/g' > "/tmp/install-config.yaml"
+  cp "/tmp/install-config.yaml" "${SHARED_DIR}/install-config.yaml"
+else
 cat >> "${SHARED_DIR}/install-config.yaml" << EOF
 networking:
   networkType: OVNKubernetes
 EOF
+fi
```

Comment:
@rvanderp3 based on some other ci operator steps (like ipi-aws) could we simplify this whole block to:

```
# TODO: move to image
curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq

touch "${SHARED_DIR}/install-config.yaml"
/tmp/yq w -i "${SHARED_DIR}/install-config.yaml" 'networking.networkType' OVNKubernetes
```

and call it a day?